### DISCUSSION TITLE
1.21 support?

### DISCUSSION THREAD

**Discussion by H-p-Singh** [BeansGalaxy/Beans-Backpacks-2#199] [Feedback]
will we get 1.21 support

**Comment by BeansGalaxy** [comment]
Yes but the mod has gone through some serious changes. It's not stable enough for survival yet but here's a ss of the creative screen so far.

![2024-09-10_00 21 28](https://github.com/user-attachments/assets/66bfd256-1672-492b-892e-e3ea7099c141)

**Comment by BeansGalaxy** [comment]
I've released the first working versions of the 1.21 mod here: https://modrinth.com/mod/beans-backpacks-3 

For now there's only Fabric support because Forge ***REALLY*** does not like 1.21 and I haven't figured out NeoForge just yet. 
Think of it as a completely different mod for now. I haven't added any support to convert from 1.20 and data and saves are subject to change until a beta release. :)

**Comment by H-p-Singh** [comment]
Will there be a forge version for 1.21 ? Or no
